src/options.c

```diff
@@ -85,6 +85,8 @@ struct ScrotOptions opt = {
     .lineMode = LINE_MODE_S_CLASSIC,
     .stackDirection = HORIZONTAL,
     .monitor = -1,
+    .thumbWorP = -1,
```

Comment:
Mostly a style nit: but I'd prefer two different variables instead of overloading one.

Feel free to ignore if doing so is too much hassle.

Reply:
I think it's a good idea that the code should be obvious instead of unobvious and commented, I'll write a version that uses a separate variable and push it if it's simpler.

Reply:
Any update on this? This can be done a later time, it's not as important as restoring previous functionality.

Also I'm thinking that the style related change should be done on a separate commit (or PR).